iron-list.html

```diff
@@ -1349,16 +1349,26 @@
         this._scrollHeight = this._estScrollHeight;
       }
     },
+
     /**
      * Scroll to a specific item in the virtual list regardless
      * of the physical items in the DOM tree.
      *
-     * @method scrollToIndex
-     * @param {number} idx The index of the item
+     * @method scrollToItem
+     * @param {(Object|number)} item The item object or its index
      */
-    scrollToIndex: function(idx) {
-      if (typeof idx !== 'number') {
-        return;
+    scrollToItem: function(item){
+      var idx = null;
+
+      if (typeof item === 'number') {
+        idx = item;
+      } else {
+        idx = this.items.indexOf(item);
+      }
+
+      //check if item is in the list
+      if(typeof idx === 'undefined' || idx < 0 || idx > this.items.length - 1){
+        return false;

```

Comment:
But right now it doesn't return anything, so this would break the contract per-se.